lib/addons/backstage-addon/backstage-values.ts

```diff
@@ -1,166 +1,166 @@
 export interface BackstageProps {
```

Comment:
I see the pattern adopted here is different to e.g. ArgoCDAddOnProps or https://catalog.workshops.aws/eks-blueprints-for-cdk/en-US/030-single-eks-cluster/033-addons/4-creating-your-own-add-ons
where "values" is a free dictionary passing through values transparently to the chart. In here differently, the interface to the chart values is structurally defined. Might that be more time consuming to maintain if downstream helm chart values are updated?